homeassistant/components/homely/homely_device.py

```diff
@@ -0,0 +1,53 @@
+"""Classes to contain the entity device link provided by Homely."""
+import logging
+
+from homelypy.devices import Device
+
+from homeassistant.helpers.entity import DeviceInfo
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class HomelyDevice:
```

Comment:
This class doesn't really seem needed. It's just proxying the same attributes as the library device exposes.